components/prism-log.js

```diff
@@ -83,7 +83,7 @@ Prism.languages.log = {
 
 	'date': {
 		pattern: RegExp(
-			/\b\d{4}[-/]\d{2}[-/]\d{2}T(?=\d{1,2}:)/.source +
+			/\b\d{4}[-/]\d{2}[-/]\d{2}(?=[T\s]\d{1,2}:)/.source +
```

Comment:
The `T` is part of the date.

```suggestion
			/\b\d{4}[-/]\d{2}[-/]\d{2}(?:T(?=\d{1,2}:)|(?=\s\d{1,2}:))/.source +
```